index.html

```diff
@@ -14,6 +16,179 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <div id="wrapper">
+      <!-- <a href="" class="mobile-nav-toggle">Toggle navigation</a> -->
+
+      <aside>
+        <div class="text-center">
+          <div class="author-image-container">
+            <!-- <img src="images/about.jp" alt=""> -->
+          </div>
+          <h1 class="author-logo"><a href="#">Nivea Perona</a></h1>
+          <span class="career"><a href="">Software Developer</a> in Madagascar</span>
+        </div>
+        <nav class="navbar">
+          <ul>
+            <li><a href="#" class="navbar-links">Home</a></li>
+            <li><a href="#about" class="navbar-links">about</a></li>
+            <li><a href="#portfolio" class="navbar-links">Portfolio</a></li>
+            <li><a href="#contact" class="navbar-links">Contact</a></li>
+          </ul>
+        </nav>
+
+        <footer class="portfolio-footer">
```

Comment:
Remember to put the footer at the end. Someone should be able to read & understand your website even if CSS is disable :)

Reply:
The footer is actually at the bottom of the page but inside the aside section. Since the aside section is at the top of the page it makes it look like the footer isn't where it's suppose to be.